### PR TITLE
sensors: adxl372: Fixup SPI CS handling

### PR DESCRIPTION
**Pull request by galak**
Replace undefined Kconfig SPI CS defines with DTS based defines.

Signed-off-by: Kumar Gala <kumar.gala@linaro.org>

### CODE REVIEW

File: drivers/sensor/adxl372/adxl372.h [hunk 1]
```diff
@@ -282,11 +282,8 @@ struct adxl372_data {
 	struct device *bus;
 #ifdef CONFIG_ADXL372_SPI
 	struct spi_config spi_cfg;
-#endif
-#if defined(CONFIG_ADXL372_SPI_GPIO_CS)
```
Comment (MaureenHelm) [inline drivers/sensor/adxl372/adxl372.h]: Why did you remove this instead of replace it with `DT_ADI_ADXL372_0_CS_GPIO_CONTROLLER`? Same question below on line 322

Comment (galak) [reply drivers/sensor/adxl372/adxl372.h]: I removed it thinking that one day we might support multiple sensors and than you'd need it always defined.  I'm happy to keep the ifdef for now.

Comment (MaureenHelm) [reply drivers/sensor/adxl372/adxl372.h]: Not necessarily. You wouldn't need it if you had multiple instances and both used hardware-supported CS. Let's keep the ifdef for now.

Comment (galak) [reply drivers/sensor/adxl372/adxl372.h]: Updated to put ifdef back.

**Comment by galak** [comment]
@MaureenHelm please review again, put the ifdef back for the CS.

File: drivers/sensor/adxl372/adxl372.h [hunk 2]
```diff
@@ -319,10 +318,8 @@ struct adxl372_dev_config {
 	const char *spi_port;
 	u16_t spi_slave;
 	u32_t spi_max_frequency;
-#ifdef CONFIG_ADXL372_SPI_GPIO_CS
```
Comment (MaureenHelm) [inline drivers/sensor/adxl372/adxl372.h]: This should be replaced with the new macro, not removed

**Review by MaureenHelm — CHANGES_REQUESTED** [review]

File: drivers/sensor/adxl372/adxl372.h (hunk 2 above)
Comment (galak) [reply drivers/sensor/adxl372/adxl372.h]: oh, missed that one.  Fixed now.

**Comment by galak** [comment]
Ok, now hopefully this is clean.  @MaureenHelm one more time.

**Review by MaureenHelm — APPROVED** [review]
